### PR TITLE
Feature-profile service

### PR DESCRIPTION
**Pull request by Kang-Quan**
Summary:
Created crud API for feature-profile-service. Tested with postman and also verified (express server is local) with database hosted on the server.

### CODE REVIEW

**Comment by huizhuansam** [comment]
@Kang-Quan please dockerize your service and add into the root docker compose

**Review by huizhuansam — CHANGES_REQUESTED** [review]
@Kang-Quan please dockerize your service and add to the root docker-compose

File: services/profile-service/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		// await AppDataSource.manager.save(user);
+		// console.log("Saved a new user with id: " + user.user_id);
+
+		// console.log("Loading users from the database...");
+		// const users = await AppDataSource.manager.find(User);
+		// console.log("Loaded users: ", users);
+
+		// console.log(
+		// 	"Here you can setup and run express / fastify / any other framework."
+		// );
```
Comment (huizhuansam) [inline services/profile-service/src/index.ts]: Code quality: remove dead code

File: services/profile-service/src/controller/user-controller.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import { Request, Response } from "express";
+import { ResponseUtil } from "../utils/Response";
+import { User } from "../entity/User";
+import { AppDataSource } from "../data-source";
+
+export class UserController {
+	// async getAuthors(req: Request, res: Response): Promise<Response> {
+	//   const authors = await AppDataSource.getRepository(User).find();
+	//   return ResponseUtil.sendResponse(res, authors, 200);
+	// }
```
Comment (huizhuansam) [inline services/profile-service/src/controller/user-controller.ts]: Code quality: remove dead code

File: services/profile-service/src/app.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import express, { Express, NextFunction, Request, Response } from "express";
+import "reflect-metadata";
+import userRoutes from "./router/user-routes";
+
+const app = express();
+
+app.use(cors());
+app.use(bodyParser.json());
+
+app.use("/user", userRoutes);
```
Comment (huizhuansam) [inline services/profile-service/src/app.ts]: Might need input from @Acerizm , to use plural nouns `/users`?

**Review by huizhuansam — DISMISSED** [review]

File: services/profile-service/src/app.ts (hunk 3 above)
Comment (Acerizm) [reply services/profile-service/src/app.ts]: sure plural nouns would be better

Comment (Kang-Quan) [reply services/profile-service/src/app.ts]: I have changed it to plural

File: services/profile-service/src/index.ts (hunk 1 above)
Comment (Kang-Quan) [reply services/profile-service/src/index.ts]: Removed dead code

**Review by Acerizm — APPROVED** [review]
LGTM! Thank you for creating the question service APIs!

**Pull request merged**